tools/firtool/firtool.cpp

```diff
@@ -91,6 +91,12 @@ static cl::opt<bool>
                        cl::desc("ignore the @info locations in the .fir file"),
                        cl::init(false));
 
+static cl::opt<std::string> blackBoxResourcePath(
+    "blackbox-resource-path",
```

Comment:
I think this can be declares as a pass option, which would be more self contained than this.

Reply:
It already is, but I can remove this option on firtool altogether, since it's more on the esoteric side. If someone then wants to resolve the weird Java class resource paths, they can just add an option to the pass (that requires specifying a custom pass pipeline, right?).